projects/aws/upgrader/Makefile

```diff
@@ -34,6 +45,36 @@ PROJECT_DEPENDENCIES=eksd/kubernetes/client eksd/kubernetes/server eksd/cni-plug
 
 include $(BASE_DIRECTORY)/Common.mk
 
+TOOLS_BIN_DIR := $(shell pwd)/hack/tools/bin
```

Comment:
move these 3 lines above the include with the rest of the "vars".  remove the spaces around the :=. instead of shell pwd use $(MAKE_ROOT) or something like that